core/templates/dev/head/services/date-time-format.service.spec.ts

```diff
@@ -62,4 +62,18 @@ describe('datetimeformatter', () => {
       df.getLocaleAbbreviatedDatetimeString(
         NOW_MILLIS - 365 * 24 * 60 * 60 * 1000)).toBe('11/21/13');
   });
+
+  it('should provide correct date format MM/DD/YYY string', () => {
+    // The implementation is being tested instead of the interface because
```

Comment:
Here's a suggestion for a clearer message. I think it's important to point out that the test is not ideal in its current form so that other devs don't just treat it as a "model solution" and copy its approach everywhere.

----------------

Note to developers: This test is not ideal, because it tests the implementation rather than the interface. However, we have not found a way to retrieve the browser current locale that is used on Date methods. Since each user can have different date formats in their browser, this makes it tricky to test the returned day, month and year of df.getLocaleDateString, which is why toLocaleDateString() needs to be computed in the expected value of the test as well.

Reply:
Done!